Dockerfile

```diff
@@ -2,4 +2,4 @@ ARG UPSTREAM_VERSION
 FROM gcr.io/hoprassociation/hoprd:${UPSTREAM_VERSION}
 
 ENV AVADO=true
-ENTRYPOINT ["/usr/bin/tini", "--", "node", "./lib/main.cjs"]
+ENTRYPOINT ["/usr/bin/tini", "--", "node", "--experimental-wasm-reftypes", "./lib/main.cjs"]
```

Comment:
Check if this could be avoided in a similar way we're avoiding the experimental wasm modules flag.

Reply:
Unfortunately not, it has to be passed directly as a parameter to the node binary.